### DISCUSSION TITLE
pt_BR iso file

### DISCUSSION THREAD

**Discussion by carlosxfelipe** [Anduin2017/AnduinOS#7] [Ideas]
Hello friends, I saw that there is a ISO version with European Portuguese, but not Brazilian Portuguese, while Portugal has 10 million speakers, Brazil has more than 215

**Comment by Anduin2017** [comment]
Hi, carlosxfelipe

Hello. Thank you for reaching out. Your feedback is very important to me. Unfortunately, I’m not very familiar with non-Asian languages and cultures. As a result, in the first wave of supported languages:

> en_US, zh_CN, zh_TW, zh_HK, ja_JP, ko_KR, vi_VN, th_TH, de_DE, fr_FR, es_ES, ru_RU, it_IT, pt_PT, ar_SA, nl_NL, sv_SE, pl_PL, tr_TR

pt_BR was not included. However, I believe adding it won’t be too difficult. (I may need to rely on some AI to help me complete some localization patches.)

Your suggestion is very valid. I will include the pt_BR version as soon as possible in the next release (1.0.1, expected to be compiled on September 15th).

Thanks again!

------------

Oi, carlosxfelipe

Olá. Obrigado por entrar em contato. Seu feedback é muito importante para mim. Infelizmente, não estou muito familiarizado com as línguas e culturas fora da Ásia. Por isso, na primeira lista de idiomas suportados:

> en_US, zh_CN, zh_TW, zh_HK, ja_JP, ko_KR, vi_VN, th_TH, de_DE, fr_FR, es_ES, ru_RU, it_IT, pt_PT, ar_SA, nl_NL, sv_SE, pl_PL, tr_TR

pt_BR não foi incluído. No entanto, acredito que adicioná-lo não será muito difícil. (Talvez eu precise contar com alguma ajuda de IA para concluir alguns patches de localização.)

Sua sugestão é muito válida. Eu vou incluir a versão pt_BR o mais rápido possível no próximo lançamento (1.0.1, previsto para ser compilado no dia 15 de setembro).

Mais uma vez, obrigado!

**Comment by rauldipeas** [comment]
HueHueBR should be present everywhere 🤣

**Comment by Anduin2017** [comment]
Thanks for your feedback!

With AnduinOS 1.0.1, pt_BR is supported.

Please download the latest image at [anduinos.com](https://www.anduinos.com)
